tests/setuptools_golang_test.py

```diff
@@ -145,6 +145,21 @@ def test_integration_internal_imports(venv):
     assert out == 'ohai, Anthony\n'
 
 
+def test_integration_user_gopath(venv, tmpdir):
+    testdir = os.path.join('testing', 'gomodules')
+
+    env = os.environ.copy()
+    env = {**os.environ, 'SETUPTOOLS_GOLANG_GOPATH': tmpdir}
```

Comment:
looks like windows is unhappy that this is pathlike

maybe:
```python
gopath = str(tmpdir.join('gopath'))
env = {..., : gopath}
```

Reply:
Thanks, I'm trying to get an env to run on windows so it might take me a couple more attempts as it's proving less straight forward than I'd hoped

Reply:
Using `$ GOPATH={shlex.quote(gopath)}` appeared to resolve the issue, thanks.